tajo-core-tests/src/test/java/org/apache/tajo/engine/eval/TestEvalTree.java

```diff
@@ -385,4 +384,34 @@ private void assertCloneEqual(EvalNode eval) throws CloneNotSupportedException {
     assertEquals(eval, copy);
     assertFalse(eval == copy);
   }
+
+  @Test
+  public final void testHashAndEqual() {
+    EvalNode e1 = new BinaryEval(EvalType.EQUAL,
+        new FieldEval("default.n1.n_nationkey", TypeFactory.create(INT4)),
+        new FieldEval("default.n2.n_nationkey", TypeFactory.create(INT4))
+    );
+
+    EvalNode e2 = new BinaryEval(EvalType.EQUAL,
+        new FieldEval("default.n1.n_name", TypeFactory.create(INT4)),
+        new FieldEval("default.n2.n_name", TypeFactory.create(INT4))
+    );
+
+    EvalNode e3 = new BinaryEval(EvalType.EQUAL,
+        new FieldEval("default.n1.n_regionkey", TypeFactory.create(INT4)),
+        new FieldEval("default.n2.n_regionkey", TypeFactory.create(INT4))
+    );
+
+    EvalNode e4 = new BinaryEval(EvalType.EQUAL,
+        new FieldEval("default.n1.n_comment", TypeFactory.create(TEXT)),
+        new FieldEval("default.n2.n_comment", TypeFactory.create(TEXT))
+    );
+
+    FieldEval f1 = new FieldEval("default.n1.n_nationkey", TypeFactory.create(INT4));
+    System.out.println(f1.hashCode());
```

Comment:
Actually, I made this test for debugging. I removed the test and inserted the comparison of hash code to TestEvalTree::assertCloneEqual, which is widely used in TestEvalTree.